remo/events/templates/view_event.jinja

```diff
@@ -301,21 +301,27 @@ Mozilla Reps - {{ event.name }}
   <!-- END Mashup code -->
 
   {% macro display_attendee(attendee) -%}
-    <div class="event-single-attendee">
-      <div class="row">
-        <div class="large-3 small-3 columns grid-profile-image">
-          <img src="{{ attendee|get_avatar_url }}"
-               class="profiles-people-avatar"
-               alt="Avatar">
-        </div>
-        <div class="large-9 small-9 columns grid-profile-text">
-          <h6>{{ attendee.get_full_name() }}</h6>
-          <p class="event-single-attendee-text">
-            {{ get_attendee_role_event(attendee, event) }}
-          </p>
+    {% if event|get_total_attendees >= 3 %}
+      <img src="{{ attendee|get_avatar_url }}"
+          class="profiles-people-avatar"
+          alt="Avatar">
```

Comment:
Minor fix: for better code readability could you indent `class` and `alt` to align with `src`? Same as you did at the rest of the template.